diagnostic.md

```diff
@@ -177,5 +210,5 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+it will be 25%
```

Comment:
137.5 px - Reason being that since it's min-width, the media query will have ```.stretch``` be 25% and at 550px width, 25% is 137.5 px.